### DISCUSSION TITLE
Thank you for your hard work -- You are appreciated!!

### DISCUSSION THREAD

**Discussion by mkgiga** [lllyasviel/stable-diffusion-webui-forge#1943] [General]
In a sea of seemingly endless demand and criticism for feature fixes, I felt compelled to make a post to show appreciation for all of the maintainers' hard work on this project. Thanks to you guys, I can get high resolution SDXL outputs on a 9 year old 980 Ti in ~30 seconds. You are awesome!!

❤️❤️❤️

**Comment by tazztone** [comment]
devs be like:
Processing request. Positive sentiment detected. Acknowledged. Continuing operations as scheduled.
